site/_resources/upgrading.md

```diff
@@ -5,126 +5,122 @@ layout: page
 
 This document describes the changes needed to upgrade your Contour installation.
 
-## Upgrading Contour 1.0.0-rc.1 to 1.0.0-rc.2
+# Upgrading Contour 0.15.3 to 1.0.0
 
-Contour 1.0.0-rc.2 is the second release candidate for Contour 1.0.
+Contour 1.0.0 is the current stable release.
 
-### Release candidate
+<div class="alert-deprecation">
```

Comment:
Nit: in the future, please follow the one line per sentence pattern. It helps to be able to comment on a specific sentence and rewriting individual sentences tends to have a lower diff count using this pattern.

Reply:
I updated the diff to do this, and added a comment for future editors.